runner/src/main/java/org/ananas/runner/steprunner/jdbc/mysql/MySQLDataTypes.java

```diff
@@ -34,20 +35,25 @@
   SMALLFLOAT("smallfloat", FieldType.FLOAT, false),
 
   DOUBLE("double", FieldType.DOUBLE, true),
-  DOUBLE_PRECISION("double precision", FieldType.DOUBLE, true),
+  DOUBLE_PRECISION("double precision", FieldType.DOUBLE, false),
 
   BOOLEAN("boolean", FieldType.BOOLEAN, true),
   BOOL("bool", FieldType.BOOLEAN, false),
 
-  BYTES("bytes", FieldType.BYTE, true),
-  BINARY("binary", FieldType.BYTE, false),
-  BYTE("byte", FieldType.BYTE, false),
-
   TEXT("text", FieldType.STRING, true),
+  BYTES("bytes", FieldType.STRING, false),
+  BINARY("binary", FieldType.STRING, false),
+  VARBINARY("varbinary", FieldType.STRING, false),
+  BYTE("byte", FieldType.INT16, false),
+  BLOB("blob", FieldType.STRING, false),
+  TINYBLOB("tinyblob", FieldType.STRING, false),
+  MEDIUMBLOB("mediumblob", FieldType.STRING, false),
+  LONGBLOB("longblob", FieldType.STRING, false),
+  MEDIUMTEXT("mediumtext", FieldType.STRING, false),
   LONGTEXT("longtext", FieldType.STRING, false),
   TINYTEXT("tinytext", FieldType.STRING, false),
   STRING("string", FieldType.STRING, false),
-  BIT("bit", FieldType.STRING, false),
+  BIT("bit", FieldType.INT32, false),
```

Comment:
please move this to INT32 section, so that we can easily find the INT32 mapping

Reply:
It's moved

Reply:
@bhou Please merge it to master :)